docs/modules/ROOT/examples/operator/bundle.yaml

```
@@ -98,6 +95,109 @@ status:
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: ManagementCenterSpec defines the desired state of ManagementCenter.
+            properties:
+              externalConnectivity:
+                description: Configuration to expose Management Center to outside.
+                properties:
+                  type:
+                    default: LoadBalancer
+                    description: 'How Management Center is exposed. Valid values are: - "ClusterIP" - "NodePort" - "Loadbalancer" (default)'
+                    enum:
+                    - ClusterIP
+                    - NodePort
+                    - LoadBalancer
+                    type: string
+                type: object
+              hazelcastClusters:
+                description: Connection configuration for the Hazelcast clusters that Management Center will monitor.
+                items:
+                  properties:
+                    address:
+                      description: IP address or DNS name of the Hazelcast cluster. If the cluster is exposed with a service name in a different namespace, use the following syntax "<service-name>.<service-namespace>".
+                      type: string
+                    name:
+                      default: dev
+                      description: Name of the Hazelcast cluster that Management Center will connect to, default is dev.
+                      type: string
+                  required:
+                  - address
+                  type: object
+                type: array
+              licenseKeySecret:
+                default: hazelcast-license-key
+                description: Name of the secret with Hazelcast Enterprise License Key.
+                type: string
+              persistence:
+                description: Configuration for Management Center persistence.
+                properties:
+                  enabled:
+                    default: true
+                    type: boolean
+                  size:
+                    anyOf:
+                    - type: integer
+                    - type: string
+                    default: 10Gi
+                    pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
+                    x-kubernetes-int-or-string: true
+                  storageClass:
+                    type: string
+                type: object
+              repository:
+                default: docker.io/hazelcast/management-center
+                description: Repository to pull the Management Center image from.
+                type: string
+              version:
+                default: 5.0
```

```suggestion
                default: '5.0'
```
When I try with the existing config it failed with error below:
```
The CustomResourceDefinition "managementcenters.hazelcast.com" is invalid: spec.validation.openAPIV3Schema.properties[spec].properties[version].default: Invalid value: "integer":  in body must be of type string: "integer"
```